schema.sql

```diff
@@ -6,6 +6,41 @@ CREATE TABLE animals
     Escape_attempts int,
     Neutered boolean,
     Weight_kg decimal,
+    PRIMARY KEY (ID),
 );
 
 ALTER TABLE animals ADD Species varchar(100);
+
+ALTER TABLE animals DROP COLUMN ID;
+ALTER TABLE animals ADD ID SERIAL PRIMARY KEY;
+
+CREATE TABLE owners
+(
+    ID int,
+    full_name VARCHAR(250),
+    age INT,
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE species
+(
+    ID int,
+    full_name VARCHAR(100),
+    PRIMARY KEY (ID)
+);
+
+ALTER TABLE animals DROP COLUMN species;
+SELECT *
+FROM animals;
+
+ALTER TABLE animals ADD species_id INT;
+SELECT *
+FROM animals;
```

Comment:
- [ ] Please remove this query because this file is responsible for your database schema. :+1: